aries-site/src/data/structures/components.js

```diff
@@ -1147,6 +1147,7 @@ export const components = [
   },
   {
     name: 'Main',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
I believe
```
If we don't pass AAA, then we say 'Passed WCAG 2.2 AA', but if we do meet AAA then we can say 'Passed WCAG 2.2 AAA'
```
so this would be passed wcag 2.2 AAA
@jcfilben

Reply:
Yes this should be 'Passed WCAG 2.2 AAA'